.github/workflows/devcontainer.yml

```diff
@@ -0,0 +1,84 @@
+name: Create dev container
```

Comment:
add a decontainer.json in a .devcontainer folder that points to the devcontainer on quay.io?

Reply:
and maybe also expose the 3030 port for the frontend because I guess even if we use the same devcontainer for the frontend it will still be two different containers(?)

Reply:
[546bc6a](https://github.com/geo-engine/geoengine-container/pull/28/commits/546bc6a2d01eb9ed065cf6cd157218972f8509dc)

[3c35dc8](https://github.com/geo-engine/geoengine-container/pull/28/commits/3c35dc837cc68817c91ffea35da9a01109c22034)

Reply:
maybe the devcontainer file should point to quay.io for the image and not a local image